dist/js/autocomplete.js

```diff
@@ -496,7 +505,8 @@
 			.param('anything', 'boolean', o, true, input)
 			.param('comma', 'boolean', o, false, input)
 			.param('notags', 'boolean', o, false, input)
-			.param('chars', 'number', o, 1, input);
+			.param('chars', 'number', o, 1, input)
+			.param('searchkeys', 'string', o, '["value", "label"]', input);
```

Comment:
I would change this to a comma separated string. So: "value,label" instead of a JSON string array. Then, you'll just remove white space and split on ",". check out my comment on searchkeys for the logic.